app/src/main/java/com/kodeco/android/coordplot/Map.kt

```diff
@@ -0,0 +1,42 @@
+package com.kodeco.android.coordplot
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kodeco.android.coordplot.ui.theme.MyApplicationTheme
+
+@Composable
+fun Map(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
+    Box(
+        modifier = Modifier
```

Comment:
```suggestion
        modifier = Modifier
```
The "modifier" argument which is passed to Map function is to configure outer Box composable.  Since you have used 'modifier = Modifier', it's not using the modifier which is passed in the function, so in this case, it is not possible to apply the modifier configurations to the outer Box composable if passed any through the Map function as arguments.

correct usage:
    Box(
        modifier = modifier

Also, if any variable or method is unused, Android studio makes the variable name or method name "grey" indicating that it is unused. :]

Reply:
got it, i will remove it from parameters since it is not used